PyInstaller/hooks/hook-shapely.py

```diff
@@ -21,7 +21,12 @@
 if compat.is_win:
     binaries = []
     lib_dir = os.path.join(pkg_dir, 'DLLs')
-    binaries += [(os.path.join(lib_dir, f), '') for f in os.listdir(lib_dir)]
+    if os.path.isdir(lib_dir):
+        binaries += [(os.path.join(lib_dir, f), '') for f in os.listdir(lib_dir)]
+    else:
+        # conda-forge package
+        lib_dir = os.path.join(pkg_dir, '..', '..', '..', '..', 'Library', 'bin')
```

Comment:
This looks like it could be used in other hooks, too. If so, please add a function to PyInstaller/utils/hooks/__init__.py. 

This path points to, lib/python/site-package/shaplely -> Library/bin, right? So can it be based on `sys.prefix`?

Further to avoid confusion, I'd add a log-message like `$lib_dir not found, trying conda-forge package $conda_lib_dir".